### PR TITLE
Add email form / replace mailchimp to sendgrid

### PR DESCRIPTION
**Pull request by antho1404**
@EmmanuelDrouin can you take it from here to design the `components/Subscribe.vue` component

### CODE REVIEW

**Comment by EmmanuelDrouin** [comment]
I Designed most of the submit form but i have some issues with the responsive and the components Button @antho1404

File: components/Subscribe.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<template>
+  <form @submit.prevent="submit">
+    <output>{{error}}</output>
+    <output>{{response}}</output>
+    <input type="email" v-model="email" name="sg_email" placeholder="you@example.com" class="field" required>
+    <Button type="submit" :value="title" class="btn" primary/>
```
Comment (antho1404) [inline components/Subscribe.vue]:
```suggestion
    <Button type="submit" class="btn" primary>{{ title }}</Button>
```

**Comment by antho1404** [comment]
fixed the text in the button, will fix the logic but can you fix the button on some responsive 
screen shot 2019-02-05 at 5 45 47 pm

**Comment by EmmanuelDrouin** [comment]
Done i fixed the button in responsive

**Comment by EmmanuelDrouin** [comment]
Can you find a way to adjust the position (center in responsive) of the form? I tried but it's not working for me 

screenshot 2019-02-05 at 19 02 58

File: components/Subscribe.vue [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    },
+    token: {
+      type: String,
+      default: '32bdd13cbff3931061eb3eca01321d84'
+    }
+  },
+  data() {
+    return {
+      email: '',
+      error: null,
```
Comment (antho1404) [inline components/Subscribe.vue]: replace with `error: "my error x"` to test the error

File: components/Subscribe.vue [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      response: null
```
Comment (antho1404) [inline components/Subscribe.vue]: replace with `response: "my response x"` to test the response

**Comment by antho1404** [comment]
done, still the `error` and `response` message to design

**Pull request merged**